verification/io/router.gobra

```diff
@@ -22,34 +22,89 @@ package io
 
 // This function corresponds to function 'core' in the IO spec
 ghost
+requires dp.Valid()
 decreases
-pure func (dp DataPlaneSpec) core() set[IO_as]
+pure func (dp DataPlaneSpec) core() set[IO_as] {
+	return dp.GetCoreAS()
+}
```

Comment:
this function is redundant. I would rename `GetCoreAS()` to `Core()` and just use that everywhere

Reply:
I agree. However, `GetCoreAS()` is a function we defined for our `DataplaneSpec` while `core()` is the original function from Isabelle. I thought it would be cleaner this way. Should I remove it?

Reply:
I would perform the change I suggested, and put a comment saying that, in the Isabelle formalisation, this is called `core`. The reason is that, for every function, we have to check well-formedness and verify it and so on. Even though the proof obligations for this function are simple, these things compound.